<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateItemDialog</class>
 <widget class="QDialog" name="UpdateItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Item</string>
  </property>
  <widget class="QLabel" name="lblItemName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category Of Item:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>âge du véhicule</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item Image:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPoids">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadItemImage">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load Item Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Année</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capacite   de la batterie(kWh)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCapacite">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="sbAnnee">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Consomation specifique  (kWh/km)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbConsomation">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
